src/ch04/exercises.ts

```diff
@@ -8,7 +8,7 @@ example, if the function is a callback). */
 
 // 2. Is JavaScript’s arguments object typesafe? If not, what can you use instead?
 
-/* arguments is not typesafe. Instead, you should spread your parameters:
+/* arguments is not typesafe. Instead, you should rest your parameters:
```

Comment:
It's a little funky to use "rest" as a verb. How would you feel about saying something like "Instead, you should use a rest parameter"?

Reply:
It's definitely better. Updated the PR.